vip-plugins/vip-plugins.php

```diff
@@ -191,7 +191,11 @@ function wpcom_vip_option_active_plugins( $value, $option ) {
 function wpcom_vip_site_option_active_sitewide_plugins( $value, $option ) {
 	$code_plugins = wpcom_vip_get_network_filtered_loaded_plugins();
 
-	if ( ! empty( $value ) && ! empty( $code_plugins ) ) {
+	if ( empty( $value ) ) {
+		$value = array();
+	}
+
+	if ( ! empty( $code_plugins ) ) {
```

Comment:
I wonder why the `! empty( $code_plugins )` check is performed here, but is not present in  `wpcom_vip_option_active_plugins` function. The `foreach` along with an empty array won't really do anything, and the `array_merge` should also be rather fast for an empty array, imho.